nhais/nhais-env.yaml

```diff
@@ -0,0 +1,8 @@
+#Common env variables
+NHAIS_SERVER_PORT: "80"
+NHAIS_LOG_LEVEL: NOTSET
+NHAIS_MAX_RETRIES: "10"
+NHAIS_RETRY_DELAY: "0.1"
+
+#User-specific env variables
+NHAIS_OUTBOUND_QUEUE_HOST: "localhost:5672/outbound"
```

Comment:
This file should be added to `.gitignore` probably

Reply:
added

Reply:
not yet :)